Commands/CalcOpenCommand.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using G1ANT.Language;
+
+namespace G1ANT.Addon.LibreOffice
+{
+    [Command(Name = "calc.open", Tooltip = "This command creates a new instance of Calc")]
+    public class CalcOpenCommand : Command
+    {
+        public CalcOpenCommand(AbstractScripter scripter): base(scripter)
+        {
+
+        }
+        public class Arguments : CommandArguments
```

Comment:
Missing empty line. Always separate methods with at least one empty line. Please apply this rule to all classes of this solution.